package.json

```diff
@@ -34,6 +34,7 @@
     "cypress": "9.6.1",
     "eslint": "8.15.0",
     "eslint-config-next": "12.1.6",
+    "gray-matter": "4.0.3",
```

Comment:
Once we touch a new package in this OR can update the node version also? 😆
Because they have [v16.15.0](https://github.com/nodejs/node/releases/tag/v16.15.0) now

Reply:
Thanks, updated in https://github.com/nimblehq/til-web/pull/8/commits/fad7ddbad5f9d74641bd617d4de4be3ad5a3c5b4